kasa/smart/modules/brightness.py

```diff
@@ -57,3 +57,7 @@ async def set_brightness(self, brightness: int):
             )
 
         return await self.call("set_device_info", {"brightness": brightness})
+
+    async def _check_supported(self):
+        """Is the module available on this device."""
```

Comment:
Please keep the docstrings consistent for this method.

Reply:
Done